tokio/tests/rt_threaded.rs

```diff
@@ -645,18 +644,16 @@ fn test_tuning() {
             break;
         }
 
-        let counter = counter.clone();
-        let interval = interval.clone();
+        if Arc::strong_count(&interval) < 5_000 {
+            let counter = counter.clone();
+            let interval = interval.clone();
 
-        if i <= 5_000 {
```

Comment:
Because the test expects 3 "valid" measurements, with unreliable enough thread scheduling (CI), it is possible t not to observe these measurements in 5k spawned tasks.